internal/exec/engine.go

```diff
@@ -51,6 +51,8 @@ const (
 	// This variable will help to identify ignition journal messages
 	// related to the user/base config.
 	ignitionFetchedConfigMsgId = "57124006b5c94805b77ce473e92a8aeb"
+
+	needNetPath = "/etc/cmdline.d/40-ignition-neednet.conf"
```

Comment:
So... I was initially going to keep this as `/run/ignition/neednet` here and have ignition-dracut have the glue code which converts that to a dracut `cmdline.d drop-in. Though... I think consensus is that ignition-dracut will be merged into this repo at some point anyway. We can still do this though if preferred.

Reply:
I'd prefer to have that indirection, even if we merge the two repos.  I won't insist though.

Reply:
Done!